.github/CODEOWNERS

```
@@ -1,6 +1,24 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
-*          @timneutkens @ijjk @shuding @styfle @huozhi @padmaia
+*          @timneutkens @ijjk @shuding @huozhi
+/.github/  @timneutkens @ijjk @shuding @styfle @huozhi @padmaia
 /docs/     @timneutkens @ijjk @shuding @styfle @huozhi @padmaia @leerob @lfades @molebox
 /examples/ @timneutkens @ijjk @shuding @leerob @lfades
+
+# SWC Build (@padmaia)
+
+/packages/next/build/   @timneutkens @ijjk @shuding @padmaia
```

```suggestion
/packages/next/build/   @timneutkens @ijjk @shuding @padmaia @huozhi 
```

wanna keep receiving notifications about build changes 🙏